### ISSUE TITLE
Close button in User Info Panel is confusing

### ISSUE THREAD

**Issue by anoadragon453** [vector-im/element-web#16316]
### Description

Video demonstrating the issue: https://streamable.com/214yjs

![image](https://user-images.githubusercontent.com/1342360/106308247-56edc800-6258-11eb-8486-34874480ad61.png)

The close button (top-right X) will close the panel in some situations:

* Room sidepanel (that let's you select room settings, member list, etc)
* File Panel
* Member list

And will go back to the previous page in others:

* UserInfo panel
* Verify Encryption (big avatar, waiting for other user to accept verification)

![image](https://user-images.githubusercontent.com/1342360/106308449-9fa58100-6258-11eb-8f3a-b45fbca52d8d.png)

I assume the back button should be used to transition from the Verify -> UserInfo screens, while the close button will close the whole panel. I also expect the close button to close the whole panel while on the UserInfo screen.

Basically it looks like the Close button has accidentally taken over the job of the Back button in the UserInfo panel.

Describe here the problem that you are experiencing, or the feature you are requesting.

---

Logs being sent: no

### Version information

- **Platform**: desktop
- **OS**: Debian Linux
- **Version**: 1.7.18

**Comment by SimonBrandner** [comment]
I can look into this. What is the desired behaviour here. The close button should always close the panel, right?

**Comment by anoadragon453** [comment]
@SimonBrandner Yes IMO.
